spot_driver/spot_driver/spot_ros2.py

```diff
@@ -355,9 +363,13 @@ def __init__(self, parameter_list: Optional[typing.List[Parameter]] = None) -> N
             if not self.spot_wrapper.is_valid:
                 return
 
-            try:
-                self.spot_cam_wrapper = SpotCamWrapper(self.ip, self.username, self.password, self.cam_logger)
-            except SystemError:
+            enable_payload_cam: Parameter = self.get_parameter("enable_payload_cam").value
+            if enable_payload_cam:
+                try:
```

Comment:
Is the try/except a check for whether the robot has a spot cam? If so, can a comment be added that explains this? Also, a warning should probably be logged that says the user enabled the spot cam, but the robot does not have one.